test.py

```diff
@@ -10,13 +10,14 @@
 
```

Comment:
The new tests look good but as I said above, the helper methods you added feel more like test helpers than things we put on the core classes at this point.